examples/tab.py

```diff
@@ -0,0 +1,45 @@
+"""
+ Shows a simple tab with two pages, one margined, one not margined.
+
+"""
+
+from pylibui.core import App
+from pylibui.controls import Window, Tab, HorizontalBox, Label
+
```

Comment:
Can you please add a new line here?

> Surround top-level function and class definitions with two blank lines. 

https://www.python.org/dev/peps/pep-0008/#blank-lines

Then I can merge this, I think :tada: